docs/src/pages/docs/installation/kubernetes.mdx

```diff
@@ -94,25 +94,20 @@ postgresql:
 
 #### Dependencies
 
-You can specify pip packages to be installed before startup, e.g. to install extra database drivers:
+Install additional packages and do any other bootstrap configuration in this script For production clusters it's
```

Comment:
sorry, let's place a `.` here: `Install additional packages and do any other bootstrap configuration in this script. For production clusters it's`

`... script. For ...`

Reply:
no worries! good catch